pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -90,6 +91,15 @@ const (
 	// StateAvailable means the host can be consumed
 	StateAvailable ProvisioningState = "available"
 
+	// StateManualCleaning means we are cleaning the old configuration and set new configuration
+	StateManualCleaning ProvisioningState = "manual cleaning"
+
+	// StateManualCleaningError means we are cleaning the old configuration and set new configuration
+	StateManualCleaningError ProvisioningState = "manual cleaning error"
+
+	// StateManualCleaned means we have Cleaned the old configuration and set new configuration
+	StateManualCleaned ProvisioningState = "manual cleaned"
```

Comment:
This state is completely pointless. It just reruns the manual cleaning code again before continuing.

Reply:
Because the state of the host is `mangeable` before and after manual clean, so I need this state to distinguish.